<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1308</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>1161</width>
      <height>192</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="statusTip">
     <string notr="true"/>
    </property>
    <property name="whatsThis">
     <string notr="true"/>
    </property>
    <property name="accessibleName">
     <string notr="true"/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string notr="true">Id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Model</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Brand</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fabrication Year</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Mileage</string>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Charging Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Maximum Range</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Price</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="delete_id">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>118</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Car by ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AllCars_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>All Cars</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>118</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Car</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Model">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Model</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Brand">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Brand</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FabricationYear">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>300</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Fabrication Year</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Mileage">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Mileage</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Chargetime">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>300</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Chargetime</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Maxrange">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>300</y>
      <width>81</width>
      <height>26</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Max Range</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Price">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>300</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Price</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Brand_filter">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Brand</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Model_filter">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Model</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Mark_brand_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>118</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Search by</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Year_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>118</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter by year</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Year_filter">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>380</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Year</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Mileage_filter">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>420</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Mileage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Mileage_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>118</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Filter by mileage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Customer_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Log in as customer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Password_field">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>480</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add_favourite">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>118</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Add favourite</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Show_favourites">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>118</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Show favourites</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="favourite_id">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>260</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Manager_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Log in as manager</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Filename">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>420</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Name of input file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="File_button">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>410</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Change CSV file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Save_changes">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>210</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Save changes to table</string>
    </property>
   </widget>
   <zorder>favourite_id</zorder>
   <zorder>Manager_button</zorder>
   <zorder>Show_favourites</zorder>
   <zorder>Add_favourite</zorder>
   <zorder>Customer_button</zorder>
   <zorder>FabricationYear</zorder>
   <zorder>delete_id</zorder>
   <zorder>Price</zorder>
   <zorder>Model</zorder>
   <zorder>Brand</zorder>
   <zorder>Maxrange</zorder>
   <zorder>Chargetime</zorder>
   <zorder>Mileage</zorder>
   <zorder>DeleteButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>tableWidget</zorder>
   <zorder>AllCars_3</zorder>
   <zorder>Brand_filter</zorder>
   <zorder>Model_filter</zorder>
   <zorder>Mark_brand_button</zorder>
   <zorder>Year_button</zorder>
   <zorder>Year_filter</zorder>
   <zorder>Mileage_filter</zorder>
   <zorder>Mileage_button</zorder>
   <zorder>Password_field</zorder>
   <zorder>Filename</zorder>
   <zorder>File_button</zorder>
   <zorder>Save_changes</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1308</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
